docs/Getting-Started-Guides/On-Premises-FPGA-Getting-Started/Frequently-Asked-Questions/Frequently-Asked-Questions-Template.rst

```diff
@@ -0,0 +1,28 @@
+Getting Started with On Premises FPGA Frequently Asked Questions
+============================================================================================
+
+To streamline the set-up process, we provide fix suggestions to the following errors that might appear during the process for reference.
+
+**SSH Setup**
+  ``build bitstream`` process stuck at ::
```

Comment:
This isn't unique to just the `buildbitstream`... this happens on a lot of FireSim steps. I would generalize a bit more here.